common/lib/xmodule/xmodule/tests/test_lti20_unit.py

```diff
@@ -20,9 +20,8 @@ class LTI20RESTResultServiceTest(unittest.TestCase):
 
     def setUp(self):
         super().setUp()
-        self.system = get_test_system()
+        self.system = get_test_system(user=self.USER_STANDIN)
```

Comment:
Nit: let's define [USER_STANDIN](https://github.com/edx/edx-platform/blob/185bedc707e368c5eb4a874acdaccd8b341d30be/common/lib/xmodule/xmodule/tests/test_lti20_unit.py#L225-L226) before its first usage. It's easier to read the code this way.